### ISSUE TITLE
Token Display

### ISSUE THREAD

**Issue by cjlcams** [Berry-Pool/nami-wallet#335]
Tokens are being displayed under collectibles tab instead of assets. Tokens like cbthc, tokhun, grow, bhosky, beans, borgz, strippercoin, fire, drip, cneta, ashib, resi, biotoken, beag, spacecoin, and def.

**Comment by HeptaSean** [comment]
Nami distinguishes between Tokens and Collectibles by the existence of metadata with the key "721" in the minting transaction.

This should only be used for NFTs, but obviously a lot of token issuers misuse it for fungible tokens, so that they are put in Collectibles in Nami.

Other wallet apps circumvent that by having just a generic category for tokens or assets, fungible or non-fungible.

**Comment by cjlcams** [comment]
Oh i see. So it means this cant be fixed?

**Comment by HeptaSean** [comment]
This can only be fixed by the Nami developers taking another approach to distinguishing assets and collectibles.

You can do nothing at the moment.

**Comment by cjlcams** [comment]
Yep, thats why I opened an issue lol. Anyway thanks for the info.
